src/mjolnir/valhalla_fetch_transit.cc

```diff
@@ -263,7 +263,7 @@ std::priority_queue<weighted_tile_t> which_tiles(const ptree& pt, const std::str
                           stations_total +
                               10 /* + routes_total * 1000 + pairs_total*/}); // TODO: factor in stop
                                                                              // pairs as well
-      LOG_INFO(GraphTile::FileSuffix(tile, false, false) + " should have " + std::to_string(stations_total) +
```

Comment:
this one looks wrong too, it should be `GraphTile::FileSuffix(tile, SUFFIX_NON_COMPRESSED, false)`